nucypher/characters/lawful.py

```diff
@@ -461,6 +462,20 @@ def revoke():
         return controller
 
 
+def ensure_correct_sender(message_kit: UmbralMessageKit, enrico=None, policy_encrypting_key=None):
```

Comment:
Annotations and docstring please! unless you have near-term plans to make additional changes here.

Reply:
Done.